### DISCUSSION TITLE
Disable "Adobe Genuine Service"

### DISCUSSION THREAD

**Discussion by rubesworkshop** [farag2/Utilities#12] [Q&A]
Thank you for sharing this script. Do you happen to know of a way to deploy without Adobe Genuine Service or how to uninstall _silently_?

Andy

**Comment by farag2** [comment]
This one?

https://github.com/farag2/Adobe-Acrobat-DC-Setup/blob/537d6e614b437f1a6f0864005563181c91984a05/Adobe_Acrobat_Reader_Pro_DC.ps1#L6
